R/choose_sites.R

```diff
@@ -0,0 +1,24 @@
+#' Identify sites with sufficient data to run a metabolism model
+#' 
+#' We will need to be more flexible later about which variables are available,
+#' since the file names will be variants on the core variable names (e.g.,
+#' wtr.local, wtr.model, etc.)
+#' 
+#' @param required_vars character vector of variable names that are all required
+#'   for a site to qualify
+#' @export
+#' @examples
+#' choose_sites()
+choose_sites <- function(required_vars=c("disch","doobs","wtr")) {
```

Comment:
does this do the same thing that `powstreams::list_sites` does? if so, that should probably wrap this instead of having different logic in `powstreams`